demo/android/sdl2surface_rawfb/build.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+#
+# This script is based on SDL sources => androidbuild.sh
+#
+##
+
+SOURCES=()
+MKSOURCES=""
+CURDIR=`pwd -P`
```

Comment:
``` `pwd -P` ``` is dangerous here. I'd prefer `"$( pwd -P )"` instead :wink:.

Reply:
done  😉